### DISCUSSION TITLE
Hola!

### DISCUSSION THREAD

**Discussion by toejam34** [MahoCommerce/maho#7] [General]
Hi @fballiano 

I am enjoying the look of your new project. I especially like the name as I frequent Fuerte a lot! :) 

Can I ask if you are the sole contributor on this project? I really admire your dedication and work ethic to resolve problems but I would not want to migrate to a project with a sole contributor. I guess in time more will flock to the project?

**Comment by fballiano** [comment]
Hi @toejam34, thanks for the message.

At the moment I'm the sole contributor, @empiricocompany helped me with the logo. Consider that I didn't announce the project to the world since I wanted to have a foundation to show and I think it's now time to announce it on my social channels.

I really hope more people will join as I think we could do nice&interesting things and move at a fast pace.
I will surely welcome contributions (as I did dure the 3+ years maintaining OM), although at the moment I will be the only people in charge for final decisions, like many other FLOSS projects, to make sure we do not stall.
